saber/boss_access/boss_access.py

```diff
@@ -19,60 +19,63 @@
 import os
 import sys
 import tempfile
-import math 
+import math
 import boto3
 from intern.remote.boss import BossRemote
 from intern.resource.boss.resource import *
 import numpy as np
 from requests import HTTPError
 
+
```

Comment:
Need to update boss access with new intern code snippets. Do we also need the xbrain merge code here?